packages/compose/src/hooks/use-media-query/index.js

```diff
@@ -6,12 +6,15 @@ import { useState, useEffect } from '@wordpress/element';
 /**
  * Runs a media query and returns its value when it changes.
  *
- * @param {string} query Media Query.
+ * @param {?string} query Media Query.
```

Comment:
Nit: If it's optional, this should be:

```suggestion
 * @param {string} [query] Media Query.
```

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#nullable-undefined-and-void-types